### ISSUE TITLE
Resizing window / toggling fullscreen results in a blank window in Win64

### ISSUE THREAD

**Issue by laamaa** [laamaa/m8c#14]
![image](https://user-images.githubusercontent.com/5983254/122528836-6dd56100-d025-11eb-82a6-d5a796a86596.png)

SDL seems to forget what was on the screen when window is resized in Windows

**Comment by laamaa** [comment]
Currently a workaround exists by manually reseting display with "r" key. There should be some kind of automatic resize detection system though that would send a reset packet

**Comment by laamaa** [comment]
possible fix https://github.com/laamaa/m8c/tree/win64_window_resize_fix, would need testing

**Comment by thulard** [comment]
Hi @laamaa , I was able to build the win64_window_resize_fix branch on my Windows 10 64 machine and I'm afraid is issue is still there. The issue occur only when you go fullscreen. 
Resizing the windows (and even maximizing it) doesn't cause any problem with both version.

**Comment by thulard** [comment]
I got it working by updating the toogle_screen function in render.c. 
I'm using "Fake fullscreen" flag on SDL_SetWindowFullscreen, this will avoid the screen resolution to change  and just change the window size and remove windows decoration.

I new to Github but I created a fork with the fix with the fix: https://github.com/thulard/m8c/commit/356b7704076cebeda832afd3337a0899d59c7726

Hope this is helpful.
